ecommerce/api_test.py

```diff
@@ -289,6 +296,14 @@ def test_create_order(self):
         del dict_before['modified_at']
         assert data_before == dict_before
 
+        if has_coupon:
+            assert RedeemedCoupon.objects.count() == 1
```

Comment:
you can assert `redeemed_coupon is not None` after you call `.first()` instead of this `.count()` assertion